.travis.yml

```diff
@@ -3,10 +3,25 @@ python: 2.7
 env:
     - TOX_ENV=py26
     - TOX_ENV=py27
-    - TOX_ENV=py34
     - TOX_ENV=py35
     - TOX_ENV=flake8
+before_install:
+    - sudo rm -f /etc/boto.cfg
```

Comment:
Could you please add a comment here why you had to that please?
(instead of on the Gtihub PR description)